kqueen_ui/utils/test_filters.py

```diff
@@ -8,9 +8,11 @@
 from kqueen_ui.utils.filters import base_url, policy_handler
 
 
-@pytest.mark.parametrize('state,icon_class', CLUSTER_STATE_MAP.items())
-def test_cluster_status_icon_class(state, icon_class):
-    assert icon_class in cluster_status_icon(state)
+@pytest.mark.parametrize('cluster,icon_class', [({'metadata': {'status_message': 'Error creating server'},
+                                                  'state': k}, v) for k, v in CLUSTER_STATE_MAP.items()])
+def test_cluster_status_icon_class(cluster, icon_class):
+    assert icon_class in cluster_status_icon(cluster)
+    assert 'Error creating server' or cluster['state'] in cluster_status_icon(cluster)
```

Comment:
This check looks like `assert True or whatever` which is always True. Probably what you wanted to do is:
```
assert any(err in cluster_status_icon(cluster) for err in ['Error creating server', cluster['state']])
```

Reply:
yeap, thanks